src/store/getters.ts

```diff
@@ -1166,5 +1179,10 @@ export default {
     }
 
     return 'Edits Saved' // should never happen
+  },
+
+  /** The court order draft file number. */
+  getCourtOrderNumberText (_state, getters): string {
+    return getters.getRestoration.courtOrder?.fileNumber || ''
```

Comment:
Why return empty string? Don't we always check if "falsy"? For example, `getCourtOrderNumberTest || 'Unknown'` ?

Reply:
This method and identical logic was duplicated in two places previously.  I'm simply copying it to the store without changing the logic.

Reply:
Oh, groan, OK.